### ISSUE TITLE
accuracy reduces after converting tensorflow h5 model to json model

### ISSUE THREAD

**Issue by LMJayasundara** [tensorflow/tfjs#3886]
when I am converting the h5 model to the JSON model I can see model accuracy reduces a lot.
What is the reason for that?

Train script
```
results = model.fit(train_generator, steps_per_epoch=train_steps, epochs=500,
                              validation_data=val_generator, validation_steps=train_steps, callbacks=[es, rl])
```

convert script
```
import tensorflowjs as tfjs
tfjs.converters.save_keras_model(model, "my-model")
```

**Comment by rthadur** [comment]
@Shanmax1996 can you please provide a codepen example for us to reproduce ?

**Comment by Shanmax1996** [comment]
@rthadur okay I will do

**Comment by Shanmax1996** [comment]
[train.zip](https://github.com/tensorflow/tfjs/files/5190056/train.zip)

**Comment by gaikwadrahul8** [comment]
Hi, @LMJayasundara 

Apologize for the delayed response and I see we have updated official documentation w.r.t tfjs-converter, please refer this [link](https://github.com/tensorflow/tfjs/tree/master/tfjs-converter) and I would suggest you to please try with latest version of `@tensorflow/tfjs@4.10.0`

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFJs team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TFJs [version](https://github.com/tensorflow/tfjs/releases/tag/tfjs-v4.10.0) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tfjs/blob/master/README.md) to stay up to date with the latest developments which are happening in the Tensorflow.js space.

Thank you for your support and cooperation.
